Services/Description/ModDescriptionService.cs

```diff
@@ -509,6 +514,34 @@ private void AddGemValue(SaveAuction auction, StringBuilder builder, Dictionary<
         builder.Append($"{McColorCodes.GRAY}Gems: {McColorCodes.YELLOW}{FormatNumber(sum)} ");
     }
 
+    private void AddCoinsPerBitValue(SaveAuction auction, Sniper.Client.Model.PriceEstimate price, StringBuilder builder, IEnumerable<string> description)
+    {
+        if (price != null && price.Median != 0 && hasBitsValue(description, out int bits))
+        {
+            var prefix = price.ItemKey == price.MedianKey ? "" : "~";
+            builder.Append($"{McColorCodes.GRAY}Coins per bit: {McColorCodes.AQUA}{prefix}{FormatNumber(price.Median/bits)} ");
+        }
+
+        bool hasBitsValue(IEnumerable<string> description, out int bits)
+        {
+            bits = 1;
+            foreach(string descLine in description)
+            {
+                Match match = Regex.Match(descLine, BitsRegexPattern);
+
+                if (match.Success && match.Groups.Count > 1)
+                {
+                    string commaSanitizedMatch = match.Groups[1].Value.Replace(",", "");
+                    if (int.TryParse(commaSanitizedMatch, out bits))
+                    {
+                        return true;
+                    }
+                }
```

Comment:
```suggestion
                if (!match.Success || match.Groups.Count <= 1)
                    continue;
                string commaSanitizedMatch = match.Groups[1].Value.Replace(",", "");
                if (int.TryParse(commaSanitizedMatch, out bits))
                    return true;
```
could be shortened like this to reduce nesting